spk/detox/src/installer.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Package
+PACKAGE="detox"
+DNAME="Detox"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put mc in the PATH
+    mkdir -p /usr/local/bin
+    mkdir -p /usr/local/usr/share/detox
```

Comment:
not sure if all this is needed... have you tried just putting detox in path? like mc does (https://github.com/SynoCommunity/spksrc/blob/master/spk/mc/src/installer.sh#L24)